### PR TITLE
add new BRC mode definition QVBR

### PR DESCRIPTION
**Pull request by XinfengZhang**
QVBR is Quality-Defined Variable Bitrate, comparing to VBR it add quality factor to the paramters

Signed-off-by: XinfengZhang <carl.zhang@intel.com>

### CODE REVIEW

**Comment by mypopydev** [comment]
LGTM, but I guess need to more comments?

File: va/va.h [hunk 1]
```diff
@@ -820,6 +820,13 @@ typedef struct _VAConfigAttrib {
  *  simultaneously. And BRC would adjust accordingly. This is so called
  *  Parallel BRC. */
 #define VA_RC_PARALLEL                  0x00000200
+/** \brief Quality defined VBR
+ * Use ICQ to determine the good enough QP for each MB such that
+ * good enough quality can be obtained without waste of bits
+ * for this BRC mode, you must set all legacy VBR parameters
+ * and ICQ_quality_factor in \c VAEncMiscParameterRateControl
+ * */
+#define VA_RC_QVBR                      0x00000400
```
Comment (mypopydev) [inline va/va.h]: In QVBR mode to setting ICQ_quality_factor is the weirdest thing, I think

Comment (mypopydev) [reply va/va.h]: I prefer to rename ICQ_quality_factor to quality_factor.

Comment (XinfengZhang) [reply va/va.h]: rename is not allowed for compatibility

Comment (XinfengZhang) [reply va/va.h]: how about add some description for reusing the definition

File: va/va.h [hunk 2]
```diff
@@ -1915,8 +1923,14 @@ typedef struct _VAEncMiscParameterRateControl
      * may exceed the target.  Ignored if set to zero.
      */
     uint32_t max_qp;
+    /** Quality factor
+     *
```
Comment (mypopydev) [inline va/va.h]: Remove empty comment line?

File: va/va.h [hunk 3]
```diff
@@ -1915,8 +1923,14 @@ typedef struct _VAEncMiscParameterRateControl
      * may exceed the target.  Ignored if set to zero.
      */
     uint32_t max_qp;
+    /** Quality factor
+     * 
+     *  the range will be different for different codec 
+     *
```
Comment (mypopydev) [inline va/va.h]: Same as above

Comment (mypopydev) [reply va/va.h]: Other part LGTM except comment empty line.

**Comment by mypopydev** [comment]
LGTM now, Thanks

**Pull request merged**